python/cudf/cudf/core/column/column.py

```diff
@@ -290,9 +296,7 @@ def from_arrow(cls, array: pa.Array) -> ColumnBase:
                 size=codes.size,
                 ordered=array.type.ordered,
             )
-        elif isinstance(
-            array.type, pd.core.arrays._arrow_utils.ArrowIntervalType
-        ):
+        elif isinstance(array.type, ArrowIntervalType):
```

Comment:
I see that cuDF has some tests involving pandas `ArrowIntervalType`, but does cuDF has public functionality that utilizes pandas `ArrowIntervalType`? I am wondering if pandas should expose this object publicly

Reply:
That would be _fantastic_

Reply:
Opened up https://github.com/pandas-dev/pandas/issues/48322 to get any thoughts but I think this should be okay to expose.